site/posts/tutorial-seminar-series.ipynb

```diff
@@ -0,0 +1,64 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Because we are unable to host an in-person tutorial for this fall season, the Xdev team has decided to switch to a virtual seminar series. We will be hosting one hour tutorial sessions scheduled for the 2nd and 4th Wednesdays of each month at 1 PM starting in October. The first tutorial session, covering how to create your first Python script, will be on October 14th. Some tutorials will be 1-off seminars on specific topics, while others will span multiple seminar sessions.\n",
+    "\n",
```

Comment:
Add a teaser marker so the entire post doesn't display on the home page:
```suggestion
    "<!-- TEASER_END -->\n",
```

Reply:
It is there on line 8 when I look at the "Files Changed" and it works when I build the site. This is confusing if it isn't there when you look.